2024/minutes/RISC-V 标准 PLIC 与平头哥玄铁 PLIC.md

```diff
@@ -0,0 +1,261 @@
+| PLIC_CTRL   | 0x01FFFFC | PLIC_CTRL      | R/W  | 0x0    | PLIC 权限控制寄存器                                                 |
+| PLIC_MTH    | 0x0200000 | PLIC_H0_MTH    | R/W  | 0x0    | 核 0 机器模式 中断阈值寄存器                                        |
+| PLIC_MCLAIM | 0x0200004 | PLIC_H0_MCLAIM | R/W  | 0x0    | 核 0 机器模式 中断响应/完成寄存器                                   |
+| -           | -         | Reserved       | -    | -      | -                                                                   |
+| PLIC_STH    | 0x0201000 | PLIC_H0_STH    | R/W  | 0x0    | 核 0 超级用户模式 中断阈值寄存器                                    |
+| PLIC_SCLAIM | 0x0201004 | PLIC_H0_SCLAIM | R/W  | 0x0    | 核 0 超级用户模式 中断响应/完成寄存器                               |
+| -           | -         | Reserved       | -    | -      | -                                                                   |
+|             | 0x0202000 | PLIC_H1_MTH    | R/W  | 0x0    | 核 1 机器模式 中断阈值寄存器                                        |
+|             | 0x0202004 | PLIC_H1_MCLAIM | R/W  | 0x0    | 核 1 机器模式 中断响应/完成寄存器                                   |
+| -           | -         | Reserved       | -    | -      | -                                                                   |
+|             | 0x0203000 | PLIC_H1_STH    | R/W  | 0x0    | 核 1 超级用户模式 中断阈值寄存器                                    |
+|             | 0x0203004 | PLIC_H1_SCLAIM | R/W  | 0x0    | 核 1 超级用户模式 中断响应/完成寄存器                               |
+| -           | -         | Reserved       | -    | -      | -                                                                   |
+|             | 0x0200000 | PLIC_Hn_MTH    | R/W  | 0x0    | 核 hart_id 机器模式 中断阈值寄存器 n=hart_id ，n<256                |
+|             | 0x0200004 | PLIC_Hn_MCLAIM | R/W  | 0x0    | 核 hart_id 机器模式 中断响应/完成寄存器 n=hart_id ，n<256           |
+| -           | -         | Reserved       | -    | -      | -                                                                   |
+|             | 0x0201000 | PLIC_Hn_STH    | R/W  | 0x0    | 核 hart_id 超级用户模式 中断阈值寄存器 n=hart_id ，n<256            |
+|             | 0x0201004 | PLIC_Hn_SCLAIM | R/W  | 0x0    | 核 hart_id 超级用户模式 中断响应/完成寄存器 n=hart_id ，n<256       |
+
+## 基础信息
+
+| 芯片系列                          | Priority 域位数 | 支持的中断源数量 | 最多支持的 hart 数量     |
+| --------------------------------- | --------------- | ---------------- | ------------------------ |
+| 平头哥 C906 PLIC                  | 5 位（0~31）    | 1023             | 1 个核（2 个上下文）     |
+| 玄铁 C920 R1S6 & C910             | 5 位（0~31）    | 1023             | 4 个核（8 个上下文）     |
+| 玄铁 C907 & C908 R1S0 & C920 R2S3 | 5 位（0~31）    | 1023             | 256 个核（512 个上下文） |
+
+## HART 与 Context ID 对应关系
+
+- **玄铁 C906**：每个 hart 有 2 个上下文，Machine Mode 和 Supervisor Mode 分别对应 Context ID 0 和 1。
+- **玄铁 C920 R1S6 & C910**：最多 4 个 hart，每个 hart 有 2 个上下文，Context ID 从 0 开始按顺序分配。
+- **玄铁 C907 & C908 R1S0 & C920 R2S3**：最多 256 个 hart，每个 hart 有 2 个上下文，Context ID 按 `2n` 和 `2n + 1` 的规则分配。
```

Comment:
For hart id `n`, M mode corresponds to `2n`, S mode corresponds to `2n + 1`

Reply:
Thanks for pointing that out! I've added the related description to clarify the details. Let me know if there's anything else I can help with：)